hydromt_wflow/workflows/soilparams.py

```diff
@@ -0,0 +1,52 @@
+"""Functions for individual soil parameters."""
+
+import numpy as np
+import xarray as xr
+
+
+def ksathorfrac(
+    da: xr.DataArray,
+    ds_like: xr.Dataset,
+    scale_method: str,
+):
+    """_summary_.
+
+    _extended_summary_
+
+    Parameters
+    ----------
+    da : xr.DataArray
+        _description_
+    ds_like : xr.Dataset
+        _description_
+    scale_method : str
+        _description_
+
+    Returns
+    -------
+    _type_
+        _description_
+    """
+    # Transfer data to a logaritmic scale
+    da.values = np.log10(da.values)
+    # Reproject the data
+    da = da.raster.reproject_like(
```

Comment:
Is there a reason why you are not using the hydromt core method hydromt.workflows.grid.grid_from_rasterdataset for the next steps ? This function includes interpolating, reprojecting and renaming. Even masking if you pass wflow_subcatch.

Reply:
Well, I basically only use the reproject method on the data on the logarithmic scale, after which is immediately goes back to a normal scale. Besides reproject nothing is needed at this stage. Therefore, in my opinion, this gives more clarity to the code.